graphql_client_web/src/lib.rs

```diff
@@ -0,0 +1,147 @@
+//! Use graphql_client inside browsers with [wasm-bindgen].
+//!
+//! This crate reexports all you need from graphql-client, so you do not need any other explicit dependencies.
+
+// #![deny(warnings)]
+#![deny(missing_docs)]
+
+#[macro_use]
+pub extern crate graphql_client;
+#[macro_use]
+extern crate wasm_bindgen;
+
+use failure::*;
+use futures::{Future, IntoFuture};
+pub use graphql_client::GraphQLQuery;
+use log::*;
+use wasm_bindgen::JsCast;
+use wasm_bindgen::JsValue;
+use wasm_bindgen_futures::JsFuture;
+
+/// The main interface to the library.
+///
+/// The workflow is the following:
+///
+/// - create a client
+/// - (optionally) configure it
+/// - use it to perform queries with the [call] method
+pub struct Client {
+    endpoint: String,
+}
+
+/// All the ways a request can go wrong.
+///
+/// not exhaustive
+#[derive(Debug, Fail)]
+#[non_exhaustive]
```

Comment:
You should add `#![feature(non_exhaustive)]`, otherwise I can't build.

Reply:
thanks, done